### PR TITLE
GNU header

### PR DESCRIPTION
**Pull request by VPhung24**

### CODE REVIEW

**Comment by VPhung24** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/snowball-tools/snowball-ts-sdk/20?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* `main`
  * **#18** <a href="https://app.graphite.dev/github/pr/snowball-tools/snowball-ts-sdk/18?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **#17** <a href="https://app.graphite.dev/github/pr/snowball-tools/snowball-ts-sdk/17?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **#19** <a href="https://app.graphite.dev/github/pr/snowball-tools/snowball-ts-sdk/19?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **#20** <a href="https://app.graphite.dev/github/pr/snowball-tools/snowball-ts-sdk/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @VPhung24 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: LICENSE [hunk 1]
```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    MPC Passkeys for EVM, Cosmos, and Solana and its uses
-    Copyright (C) 2023 Snowball Tools Inc.
+    <one line to give the program's name and a brief idea of what it does.>
```
Comment (ericlewis) [inline LICENSE:634]: ?

File: packages/helpers/src/auth.ts [hunk 2]
```diff
@@ -1,3 +1,21 @@
+/*
+ * Snowball Tools SDK, authentication, authorization, smart wallets
+ * Copyright (C) 2024  Snowball Tools Inc.
+ 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
```
Comment (ericlewis) [inline packages/helpers/src/auth.ts:8]: uhhhh. we are distributing under the GPL 3.0 license?

**Review by ericlewis — CHANGES_REQUESTED** [review]
hmmm, I think our licensing might not be correct here.

File: LICENSE (hunk 1 above)
Comment (VPhung24) [reply LICENSE:634]: [GNUv3](https://www.gnu.org/licenses/gpl-3.0.en.html) LICENSE shouldn't be edited at all. It should be left as is

File: packages/helpers/src/auth.ts (hunk 2 above)
Comment (VPhung24) [reply packages/helpers/src/auth.ts:8]: only the sdk. open to changing this

File: LICENSE [hunk 3]
```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
```
Comment (VPhung24) [inline LICENSE:631]: <i class='graphite__hidden'>[Re: lines 5 to 6]<br/><br/></i>

Reference: "Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed."<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/snowball-tools/snowball-ts-sdk/20?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Comment by VPhung24** [comment]
> hmmm, I think our licensing might not be correct here.
> 
> 

Yeah, I need to add it to the readme -- I'll make the change but here's the start of it stating to not edit the file. I could be wrong. https://app.graphite.dev/github/pr/snowball-tools/snowball-ts-sdk/20/GNU-header#comment-PRRC_kwDOKLKkeM5YIvVs

**Comment by VPhung24** [comment]
added to readme: https://app.graphite.dev/github/pr/snowball-tools/snowball-ts-sdk/20/GNU-header#file-README.md

**Review by ericlewis — APPROVED** [review]
we can always relicense I suppose

**Pull request merged**